apps/client-server/src/app/validation/validators/common-field-validators.ts

```diff
@@ -170,7 +174,7 @@ export async function validateRequiredTagField({
 
   for (const [fieldName, field] of Object.entries(fields)) {
     // Skip if field is not required or hidden
-    if (!field.required || field.hidden) continue;
+    if (!field.required || field.hidden || fieldName === 'tags') continue;
```

Comment:
q: if the function is called `validateRequiredTagField` why are we skipping tags? Doesn't this just make the function dead?

Reply:
I guess so? It was added by you so i was not sure if i can just remove it or if it does have some purpose